### ISSUE TITLE
search result page layout problem

### ISSUE THREAD

**Issue by jesseminn** [silvestreh/atom-material-ui#227]
![2016-02-24 6 44 20](https://cloud.githubusercontent.com/assets/14268416/13309051/078d0e9a-dbb4-11e5-89e6-4f9858e47e74.png)

1. i know there's an option to make scrollbar thinner or not, but it doesn't work for me. Is it possible to make scroll only visible when you start scrolling?

2. the layout of project find result page looks weird. Because of the line height and font size, i could only see 10 line in the result page. and the position of the scrollbar is weird, too.

3. not in this screenshot, but the option "font use roboto" doesn't work for me, too. it makes no difference if i check it or not.

**Comment by silvestreh** [comment]
Those options were removed, but you're still seeing them because of a cache issue with Atom.

Open the dev console and run this: `atom.config.set("atom-material-ui");`. This will reset your theme's settings. Also, remove the `~/.atom/compile-cache/` directory and restart Atom.

If those options aren't going away, then open your `~/.atom/config.cson` file and remove all preferences in `atom-material-ui` so it looks like:

```cson
"atom-material-ui":
    colors: {}
    fonts: {}
    tabs: {}
    treeView: {}
    ui: {}
```

**Comment by jesseminn** [comment]
it fix everything besides #2 the layout problem of search result page, or it's not because material theme?

**Comment by silvestreh** [comment]
Ah yes… that's not exactly a bug, but it's something can be improved. I'll take a look at it when I get some free time :smile:

**Comment by silvestreh** [comment]
Fixed in d55620a5e69a6c090609d7de430294e766de1028 :smile:
